ocaml/typing/primitive.ml

```diff
@@ -30,6 +30,7 @@ type description =
   { prim_name: string;         (* Name of primitive  or C function *)
     prim_arity: int;           (* Number of arguments *)
     prim_alloc: bool;          (* Does it allocates or raise? *)
+    prim_builtin: bool;        (* Is the compiler allowed to replace it? *)
```

Comment:
Let's extend this comment to clarify that `prim_builtin` does not apply to `%` primitives (which might also be thought of as "builtin"), but only applies when `prim_name` is that of a C function.  Maybe we could also rename the field to `prim_c_builtin`?

Reply:
I added a comment that hopefully clarifies it. There is currently no error if `[@@builtin]` is attached to % primitives, it will just be ignored. Should I add a check?

Reply:
I think it will probably be fine without the check, at least for the moment.

Reply:
great, thanks! I'll change the name and bootstrap.